### PR TITLE
LPS-168892 & LPS-168896

### PR DESCRIPTION
**Pull request by rotty3000**

### CODE REVIEW

**Comment by rotty3000** [comment]
simplified the history

**Comment by rotty3000** [comment]
ci:reopen

File: workspaces/sample-default-workspace/client-extensions/easy/src/main/resources/application.properties [hunk 1]
```diff
@@ -4,4 +4,5 @@ logging.level.org.springframework.security=DEBUG
 logging.level.org.springframework.web=DEBUG
 liferay.oauth.application.external.reference.code=easy-oauth-application-user-agent
 liferay.portal.url=http://localhost:8080
+dxp.domains=localhost
```
Comment (gamerson) [inline workspaces/sample-default-workspace/client-extensions/easy/src/main/resources/application.properties:7]: Hey Ray can you sort this?

Comment (rotty3000) [reply workspaces/sample-default-workspace/client-extensions/easy/src/main/resources/application.properties:7]: hey, this is a superfluous property so I will remove it. thx

**Pull request closed without merging**